src/main/java/com/stripe/model/MetadataStore.java

```diff
@@ -14,14 +14,10 @@
  */
 public interface MetadataStore<T> {
 	Map<String, String> getMetadata();
-	void setMetadata(Map<String, String> metadata);
 
 	MetadataStore<T> update(Map<String, Object> params) throws AuthenticationException, InvalidRequestException,
 		APIConnectionException, CardException, APIException;
 
-	@Deprecated
-	MetadataStore<T> update(Map<String, Object> params, String apiKey) throws AuthenticationException, InvalidRequestException,
```

Comment:
would rather not nuke this method yet (save it for a major release) unless it's really causing you trouble